scripts/partition_alloc/update.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+REPO_ROOT=$(pwd)
+REPO_CHROMIUM=$REPO_ROOT/external/chromium
+REPO_SRC=$REPO_CHROMIUM/src
+NEW_CHROMIUM=$REPO_ROOT/chromium
+NEW_SRC=$NEW_CHROMIUM/src
+DEPOT_TOOLS=$REPO_ROOT/depot_tools
+
+# Download depot_tools
+if [[ -d "$DEPOT_TOOLS" ]];
+then
+	  rm -rf $DEPOT_TOOLS
+fi
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
```

Comment:
I'd rather this used `$DEPOT_TOOLS` instead of coincidentally populating a matching dir.